src/theme/base/_utils.scss

```diff
@@ -0,0 +1,12 @@
+@mixin generalAlignment() {
```

Comment:
I'd prefer to use composition rather than a mixin.

A mixing will add the same code in different places, increasing the final bundle. If instead you compose a class, it will use the same class everywhere.

```
.generalAlignment {
  margin-left: auto;
    margin-right: auto;
    
    @media screen and (min-width: 768px) {
        max-width: 992px;
    }
     @media screen and (min-width: 1400px) {
        max-width: 1200px;
    }
}
```

The, where you want to use it:

```
@media screen and (min-width: 768px) {
  . sectionInner {
    composes: generalAlignment;
    // more styles for sectionInner
  }
}
```

This will result in reusing the `.generalAlignment` in the DOM element. This is automatically done by webpack ;)

```
<div class="sectionInner generalAlignment" >
```

Reply:
Great idea!

Reply:
Great idea! I did not know about the bundle size. Good to know thank you :smile: